src/index.js

```diff
@@ -0,0 +1,151 @@
+import './style.css';
+import { saveTasks, loadTasks } from './modules/storage.js';
+
+const taskList = document.getElementById('taskList');
+const newTaskForm = document.getElementById('newTaskForm');
+const clearCompletedButton = document.getElementById('clearCompleted');
+let tasks = [];
+
+const updateIndex = () => {
+  tasks.forEach((element, index) => {
+    element.index = index + 1;
+  });
+};
+
+const renderTaskList = () => {
+  taskList.innerHTML = '';
+
+  tasks.forEach((task, index) => {
+    const taskElement = document.createElement('li');
+    taskElement.className = 'task';
+    taskElement.draggable = true;
+    taskElement.dataset.index = task.index;
+
+    const taskCheckbox = document.createElement('input');
+    taskCheckbox.type = 'checkbox';
```

Comment:
- [ ] You seems to have went overboard in your project to implement future functionality. However, for this milestone, you're been tested for certain skills and   its ideal that you at least stick to that.

- [ ]  Based on project requirements, you're to  create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
- description [string].
- completed [bool].
- index: [number].

Right now this is done dynamically, which is good but a future functionality. Kindly follow the instructions to show the required skills been tested here

## Instructions
![image](https://user-images.githubusercontent.com/50990925/229676773-317164c8-db0b-45f7-ac0d-8143262db81c.png)